doc/classes/Viewport.xml

```diff
@@ -33,6 +33,18 @@
 				Returns the first valid [World3D] for this viewport, searching the [member world_3d] property of itself and any Viewport ancestor.
 			</description>
 		</method>
+		<method name="get_audio_listener_2d" qualifiers="const">
+			<return type="AudioListener2D" />
+			<description>
+				Returns the currently active 2D audio listener. Returns null if there are no active 2D audio listeners.
+			</description>
+		</method>
+		<method name="get_audio_listener_3d" qualifiers="const">
+			<return type="AudioListener3D" />
+			<description>
+				Returns the currently active 3D audio listener.
```

Comment:
We should also add a note that if this is null, the current camera is treated as the audio listener instead.

Reply:
Done